visualization/FairnessDashboard/src/Controls/ModelComparisonChart.tsx

```diff
@@ -182,9 +182,11 @@ export class ModelComparisonChart extends React.PureComponent<IModelComparisonPr
             return series;
         });
         const accuracyMetricTitle = selectedMetric.title;
```

Comment:
Github may be cutting out instances when I search, but it seems there are only two uses of this, both with the same ternary to do lowercase, so you could just set here using the ternary to correctly capitalize, then use the variable below without modification?

Reply:
done